modules/appeals_api/app/controllers/appeals_api/v2/decision_reviews/legacy_appeals_controller.rb

```diff
@@ -80,4 +81,8 @@ def render_unusable_response_error
     render status: :bad_gateway,
            json: Common::Exceptions::BackendServiceException.new(detail: 'Unusable upstream response')
   end
+
+  def enabled?
+    true if Settings.modules_appeals_api.legacy_appeals
```

Comment:
Let's return the setting value directly, since it's already a boolean

Reply:
oh yes